### PR TITLE
twitter logo updated to x

### LINKED ISSUE #8994: Twitter logo needs to be updated to X
### Description
in the footer of [docs.meshery.io](https://docs.meshery.io/) twitter's old logo is needs to be updated to X

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by joshijoe05**
This PR fixes #8994 
Update the Twitter Logo To X

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8a0e145ab891756ec0f3149281620de19a2e5b23 at: https://652237b1fcd97675e2295675--meshery-docs-preview.netlify.app

**Review by sandramsc — COMMENTED** [review]
Looks good.  Great effort!
A few items before merging:
1. Can you please address this [lint error](https://github.com/meshery/meshery/actions/runs/6445615639/job/17499868533?pr=9049).
2. For the DCO check error, you can do this

    git checkout <branch-name>
    git commit --amend -s
    git push -f origin <branch-name>

This adds the signoff to the commit you make.

**Comment by l5io** [comment]
🚀 Preview for commit c6c958802aa2a0895a23641417494562a9b22312 at: https://6523c2a1ac0ed75bab50cf0e--meshery-docs-preview.netlify.app

**Comment by saurabh100ni** [comment]
Thanks for raising the PR, let's Discuss this in the Meshery Development Call today,
Please add this as an agenda in the Doc  : [Doc](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit)

File: docs/_includes/footer.html [hunk 1]
```diff
@@ -77,7 +77,7 @@
...
+          src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAyNy41LjAsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjxzdmcgdmVyc2lvbj0iMS4xIiBpZD0ic3ZnNSIgeG1sbnM6c3ZnPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyINCgkgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeD0iMHB4IiB5PSIwcHgiIHZpZXdCb3g9IjAgMCAxNjY4LjU2IDEyMjEuMTkiDQoJIHN0eWxlPSJlbmFibGUtYmFja2dyb3VuZDpuZXcgMCAwIDE2NjguNTYgMTIyMS4xOTsiIHhtbDpzcGFjZT0icHJlc2VydmUiPg0KPGcgaWQ9ImxheWVyMSIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoNTIuMzkwMDg4LC0yNS4wNTg1OTcpIj4NCgk8cGF0aCBpZD0icGF0aDEwMDkiIGQ9Ik0yODMuOTQsMTY3LjMxbDM4Ni4zOSw1MTYuNjRMMjgxLjUsMTEwNGg4Ny41MWwzNDAuNDItMzY3Ljc2TDk4NC40OCwxMTA0aDI5Ny44TDg3NC4xNSw1NTguM2wzNjEuOTItMzkwLjk5DQoJCWgtODcuNTFsLTMxMy41MSwzMzguN2wtMjUzLjMxLTMzOC43SDI4My45NHogTTQxMi42MywyMzEuNzdoMTM2LjgxbDYwNC4xMyw4MDcuNzZoLTEzNi44MUw0MTIuNjMsMjMxLjc3eiIvPg0KPC9nPg0KPC9zdmc+DQo="
```
Comment (leecalcote) [inline docs/_includes/footer.html:80]: Please use an SVG.

**Comment by akagami-harsh** [comment]
i can see the pr is not merged , can i give it a try

**Comment by leecalcote** [comment]
@joshijoe05 checking in... are you intending to address the comments above?

**Comment by leecalcote** [comment]
@joshijoe05 checking in... are you intending to address the comments above?

**Pull request closed without merging**